modules/enabling-etcd-encryption.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * authentication/encrypting-etcd.adoc
+
+[id="enabling-etcd_{context}"]
+= Enabling etcd encryption
+
+You can enable etcd encryption to encrypt sensitive resources in your cluster.
+
+[WARNING]
+====
+Do not back up etcd until the encryption process is complete.
```

Comment:
actually this should work (@xingxingxia I assume you test this). Of course the backup then is not fully encrypted.

Reply:
Yes, it should work. This is moved to Etcd Component and pending Etcd QE Ge Liu to verify it https://bugzilla.redhat.com/show_bug.cgi?id=1776811 which needs additional Doc in DR doc section too.

Reply:
So users don't need to wait until encryption is complete to take a backup? Both upon initial enabling and during the weekly rotation?  CC @retroflexer and @geliu2016 

I'm going to remove this warning for now and merge this PR. And we can follow up on this in the PR when we add update DR.

Reply:
Thanks @xingxingxia for testing it again. The config with the failed state is active for next couple of days. @sttts and anyone else to look at it?

Reply:
@retroflexer we don't have anybody in US timezone for that topic. We need access to the cluster tomorrow (EU) morning. I also commented on the BZ to get more logs and the encryption config.

Reply:
@sttts @xingxingxia So what message do we want to present to users here? So even if it works to take a backup in the middle of the initial encryption process, if they do it and the backup isn't fully encrypted, I'm guessing that's something that we would want to warn them of.

What do you say to adding a warning along the lines of `Do not back up etcd until the encryption process is complete or the backup might not be fully encrypted`. Or `initial encryption process` if the concern is only for the first time encryption is enabled? wdyt?

Are there any concerns with taking a backup in the middle of a weekly rotation, or is that handled okay with the multiple versions kept of the keys?